apps/dapp/src/constants/env/production.tsx

```diff
@@ -3,7 +3,7 @@ import { Environment } from './types';
 
 const env: Environment = {
   alchemyId: 'XiIZxWykHU5AOFBwxKgxseXWN984Mp8F',
-  rpcUrl: 'https://eth.llamarpc.com',
+  rpcUrl: 'https://rpc.ankr.com/eth',
```

Comment:
out of interest, was llamarpc causing issues?

Reply:
Yes looks like there were some issues https://github.com/TempleDAO/temple/pull/1054#issuecomment-2211968381 so moved to this one.